mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -35,13 +35,40 @@ interface TooltipProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should tooltip be shown on click
+   */
+  isShownOnClick?: boolean;
```

Comment:
I think tooltips should be shown if you press button some time on mobile (like in github app) and should not be shown if user clicked on button

Reply:
If you will not find appropriate solution, lets remove tooltips from such interactive elements, 


hope this also could help: https://spacejelly.dev/posts/how-to-detect-long-press-gestures-in-javascript-events-in-react

Reply:
I suggest reverting my changes related to showing tooltips, and have someone create a separate task for me to handle tooltips on mobile, since the changes will affect the entire application

Reply:
I just wanted to fix this behavior on Dropdown component, when the tooltip appears and quickly disappears on click on desktops. Since it happens in all components which have tooltips, we need to fix the Tooltip component: 1) onClick behavior on desktops, 2) "onLongTouch" on mobiles

[Screencast from 09-07-24 17:45:28.webm](https://github.com/tritonJS826/masters-way/assets/8752900/a57f9b0d-f9ff-48f4-96a6-99192dae6287)